File: 8-kyu/001-even-or-odd/001-even-or-odd.js
const even_or_odd = (number) => {
   if( number % 2 === 0) {
       return 'Even'
   } else {
       return 'Odd'
   }
}

const even_or_odd = (number) => (number % 2 === 0) ? 'Even' : 'Odd'

function even_or_odd(number) {
  return ["Even", "Odd"][number % 2]
}

console.log(even_or_odd(5));

File: 8-kyu/001-even-or-odd/001-even-or-odd.dart
String evenOrOdd(int number) {
  if (number % 2 == 0) {
    return 'Even';
  } else {
    return 'Odd';
  }
}

String evenOrOdd(int number) {
  return (number % 2 == 0) ? "Even" : "Odd";
}

String evenOrOdd(int number) => (number % 2 == 0) ? "Even" : "Odd";

String evenOrOdd(int number) => number.isEven ? "Even" : "Odd";

String evenOrOdd(int number) {
  return ['Even', "Odd"][number % 2];
}

Map results = {true: "Even", false: "Odd"};

String evenOrOdd(int number) => results[number.isEven];

main() {
  print(evenOrOdd(4));
}
